addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -92,6 +92,10 @@ insertModelFields("mail.message", {
         type: "many2many",
     },
     res_model_name: { string: "Res Model Name", type: "char" },
+    user_follower_id: {
+        string: "Follower id if user is following the related record otherwise 0",
+        type: "integer", default: 0,
```

Comment:
don't forget to update here if we switch from zero to False

Reply:
As suggested, I have used required=false, instead of default=0